src/routes/dashboard/streams/[streamId=i32]/topics/[topicId=i32]/messages/+page.server.ts

```diff
@@ -0,0 +1,38 @@
+import { fetchApi } from '$lib/api/fetchApi';
+import { handleFetchErrors } from '$lib/api/handleFetchErrors';
+import { messageDetailsMapper } from '../../../../../../../lib/domain/MessageDetails';
+import { topicDetailsMapper } from '../../../../../../../lib/domain/TopicDetails';
+
+export const load = async ({ params, cookies }) => {
+  const getMessages = async () => {
+    const result = await fetchApi({
+      method: 'GET',
+      path: `/streams/${+params.streamId}/topics/${+params.topicId}/messages`,
+      cookies,
+      queryParams: {
```

Comment:
Regarding the comment: 'Requirements - Implement pagination on server side'

You can implement frontend pagination right now using the messages retrieval endpoint: 
GET {{url}}/streams/{{stream_id}}/topics/{{topic_id}}/messages?consumer_id={{consumer_id}}&partition_id={{partition_id}}&kind=offset&value=0&count=10&auto_commit=false
where offset is the position from which you want to load messages and count is the number of messages. If you go beyond the range with the offset, it will return an empty array.

Reply:
Maybe an infinite scroll would be nice for messages view?

Reply:
Infinite scroll won't work when you have lots of data - pagination rendered in UI, where you simply take the `current offset+1` as the total results available, and then render number of total pages based on this simple calculation, will do the trick.